src/jobs/microk8s_install.yml

```diff
@@ -53,6 +51,27 @@ steps:
   - run:
       name: Setup microk8s and helm
       command: |
+        microk8s_wait_ready() {
```

Comment:
we probably can reuse something like this:
https://circleci.com/orbs/registry/orb/kimh/run-with-retry
?

Reply:
I get that reusing code is better, but in this case I would rather not... We don't know who `kimh` is, and he could overwrite the orb tomorrow, for all we know, with something that breaks our builds or leaks the secrets in our environment variables.

P.S. - to `kimh`, if you happen to be reading this. Thank you for sharing your work. Nothing against you personally. It is just a matter of principle. :)

Reply:
> and he could overwrite the orb tomorrow
this is not correct, published orbs are immutable, and the code is public

Reply:
Yes, you're right, they are immutable. But, would using a command from an orb mean that I'd have to break this into steps whenever I wanted to use it? Would't that just add unnecessary complexity? Is the wait function that complex that justifies using an orb for it? Is the code from the orb better than the function in the script? Would it be easier to maintain using the orb? Would it be more legible? Would it be more stable? Would it be more succinct, given that would have to pass the commands I want to execute as an argument anyways?

Reply:
I think in this specific case using the orb and dividing into multiple steps is beneficial to readability/usability etc.

Reply:
I don’t really agree that wrapping a simple script with yaml makes it more readable or usable, but if that’s what it takes to get this merged I’ll gladly do it, because currently we don’t have nightly tests of our chart against K8s versions because this isn’t merged. I’ll submit a PR tomorrow.

Reply:
Don't wanna push this too hard, but please think about splitting this big command into several at least

Reply:
Ok, it might make watching the job running in CircleCI a bit more user friendly. Done